openmetadata-ui/src/main/resources/ui/src/components/Settings/Team/TeamImportResult/TeamImportResult.component.tsx

```diff
@@ -164,16 +164,25 @@ export const TeamImportResult = ({
         title: t('label.role-plural'),
         dataIndex: 'defaultRoles',
         key: 'defaultRoles',
+        width: 100,
         render: (role: TeamCSVRecord['defaultRoles']) => {
-          return <Typography.Paragraph>{role || '--'}</Typography.Paragraph>;
+          return (
+            <Typography.Paragraph style={{ width: 200 }}>
+              {role || '--'}
+            </Typography.Paragraph>
+          );
```

Comment:
we are giving width of 100 to column and for text we are giving width of 200, it will overflow?